--- Modules/FindCUDA.cmake	2012-06-20 19:28:13.894478383 -0700
+++ Modules-fix/FindCUDA.cmake	2012-06-20 19:29:06.734436533 -0700
@@ -539,7 +539,7 @@
     NO_DEFAULT_PATH
     )
   # Search default search paths, after we search our own set of paths.
-  find_library(${_var} NAMES ${_names} DOC ${_doc})
+  find_library(${_var} NAMES ${_names} PATH_SUFFIXES "nvidia-current" "nvidia-current-updates" DOC ${_doc})
 endmacro()
 
 # CUDA_LIBRARIES
